indicators/NO_URGR_001-006/R/NO_URGR_001-006.qmd

```diff
@@ -0,0 +1,2547 @@
+                           urgr_regional= round2(urban_green_area_m2 / total_urban_area_m2 * 100, 2)) %>%
+                    select(region_name, indicator, urgr_regional)
+
+regions$kommune_nr_2023 <- as.numeric(regions$kommune_nr_2023) 
+regions$kommune_nr_2023[regions$kommune_nr_2023 == 0301] <- 301
+urgr003_regional <- urgr003 %>%
+                    left_join(., regions, join_by(LAU_ID == kommune_nr_2023)) %>%
+                    group_by(region_name) %>%
+                    summarise(urban_green_area_m2 = sum(total_area_et_m2),
+                              total_urban_area_m2 = sum(unique(soaa_total_area_m2))) %>%
+                    mutate(indicator = rep("URGR 003", nrow(.)),
+                           urgr_regional = round2(urban_green_area_m2 / total_urban_area_m2 * 100, 2)) %>%
+                    select(region_name, indicator, urgr_regional)  
+
+
+urgr004_regional <- urgr004 %>%
+                    left_join(., regions, join_by(LAU_ID == kommune_nr_2023)) %>%
+                    group_by(region_name) %>%
+                    summarise(urban_green_area_m2 = sum(total_non_imper_area_m2),
+                              total_urban_area_m2 = sum(total_soaa_area_m2)) %>%
+                    mutate(indicator = rep("URGR 004", nrow(.)),
+                           urgr_regional = round2(urban_green_area_m2 / total_urban_area_m2 * 100, 2)) %>%
+                    select(region_name, indicator, urgr_regional)  
+
+urgr005_regional <- urgr005 %>%
+                    left_join(., regions, join_by(fylkesnumm == fylke_nr)) %>%
+                    group_by(region_name) %>%
+                    summarise(urban_green_area_m2 = sum(total_non_imper_area_m2),
+                              total_urban_area_m2 = sum(total_soaa_area_m2)) %>%
+                    mutate(indicator = rep("URGR 005", nrow(.)),
+                           urgr_regional = round2(urban_green_area_m2 / total_urban_area_m2 * 100, 2)) %>%
+                    select(region_name, indicator, urgr_regional)  
+
+urgr006_regional <- urgr006 %>%
+                    left_join(., regions, join_by(fylkesnumm == fylke_nr)) %>%
+                    group_by(region_name) %>%
+                    summarise(urban_green_area_m2 = sum(total_non_imper_area_m2),
+                              total_urban_area_m2 = sum(total_soaa_area_m2)) %>%
+                    mutate(indicator = rep("URGR 006", nrow(.)),
+                           urgr_regional = round2(urban_green_area_m2 / total_urban_area_m2 * 100, 2)) %>%
+                    select(region_name, indicator, urgr_regional) 
+
+# Create the table of outputs
+urgr_df_regional <- bind_rows(urgr001_regional,
+                              urgr002_regional,
+                              urgr003_regional,
+                              urgr004_regional,
+                              urgr005_regional,
+                              urgr006_regional)
+                    
+
```

Comment:
I was wondering how the regional table should be sorted. Is the interest in comparing the regions per indicator (e.g. indicator 1 -6 for vestlanded) or one indicator for all regions (indicator 1). Depending on what is the interest one might want to sort the table differently.

Reply:
I suggest using DT::datatable() instead of kbl(). That way one can sort manually in the interactive html table